tests/backup_restore_test.go

```diff
@@ -424,39 +513,43 @@ func doBackup(ctx context.Context, driver dbplugin.Driver, database string) (*by
 }
 
 // Concurrently update a single row to mimic the ongoing business workload.
-// Returns the timestamp after inserting the initial value so we could check the PITR is done right.
-func startUpdateRow(ctx context.Context, t *testing.T, database string, port int) int64 {
+// Returns the timestamp after inserting the initial value so we could check the PITR is done right, passes the current id value into done chan when the update is finished.
+func startUpdateRow(ctx context.Context, t *testing.T, database string, port int, createAndInit bool, done chan<- int) int64 {
```

Comment:
We can simplify this by always initializing the row to 0 before updating.

Reply:
How about this one? I mean we do not need to pass in a new parameter, but return the channel instead.

Reply:
I get the wrong meaning... Sorry.
Return the channel is good.

Reply:
SG.
BTW, let's resolve a conversation by some message, like "IMO it's not needed" so that the reviewers can make sure that their comments are properly received.

Reply:
It's OK to resolve a conversation with no message if you do as the comment says.